### PR TITLE
get first block for showing behavior; fix terrible intellij bug

### PR DESCRIPTION
**Pull request by atvaccaro**

### CODE REVIEW

File: src/main/java/minecraftbyexample/ClientOnlyProxy.java [hunk 1]
```diff
@@ -103,7 +105,7 @@ public void postInit()
     minecraftbyexample.mbe11_item_variants.StartupClientOnly.postInitClientOnly();
     minecraftbyexample.mbe12_item_nbt_animate.StartupClientOnly.postInitClientOnly();
     minecraftbyexample.mbe13_item_tools.StartupClientOnly.postInitClientOnly();
-    minecraftbyexample.mbe15_item_dynamic_item_model.StartupClientOnly.postInitClientOnly();
```
Comment (dealingwith) [inline src/main/java/minecraftbyexample/ClientOnlyProxy.java]: Was this deletion intentional?

Comment (dealingwith) [reply src/main/java/minecraftbyexample/ClientOnlyProxy.java]: I guess so 😂 (I see the additional deletions below)

File: src/main/java/minecraftbyexample/mbe01_block_simple/BlockSimple.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
  * User: The Grey Ghost
  * Date: 24/12/2014
  *
- * BlockSimple is a ordinary solid cube with the six faces numbered from 0 - 5.
+ * BlockDropper is a ordinary solid cube with the six faces numbered from 0 - 5.
```
Comment (dealingwith) [inline src/main/java/minecraftbyexample/mbe01_block_simple/BlockSimple.java]: Was this intentional?

Comment (atvaccaro) [reply src/main/java/minecraftbyexample/mbe01_block_simple/BlockSimple.java]: Nope.

File: src/main/java/minecraftbyexample/CommonProxy.java [hunk 3]
```diff
@@ -65,7 +66,7 @@ public void init()
     minecraftbyexample.mbe11_item_variants.StartupCommon.initCommon();
     minecraftbyexample.mbe12_item_nbt_animate.StartupCommon.initCommon();
     minecraftbyexample.mbe13_item_tools.StartupCommon.initCommon();
-    minecraftbyexample.mbe16_item_food.StartupCommon.initCommon();
+    minecraftbyexample.mbe17_block_crops.StartupCommon.initCommon();
```
Comment (dealingwith) [inline src/main/java/minecraftbyexample/CommonProxy.java:73]: Another unintentional deletion?

Comment (atvaccaro) [reply src/main/java/minecraftbyexample/CommonProxy.java:73]: Yeah D: fixed

File: src/main/java/minecraftbyexample/ClientOnlyProxy.java (hunk 1 above)
Comment (dealingwith) [reply src/main/java/minecraftbyexample/ClientOnlyProxy.java]: Now I'm not so sure. Going through and cleaning up this PR, I'm going to try putting this back in

Comment (dealingwith) [reply src/main/java/minecraftbyexample/ClientOnlyProxy.java]: Huh? It's still there in the source...

Comment (dealingwith) [reply src/main/java/minecraftbyexample/ClientOnlyProxy.java]: ...never mind, I was looking at the wrong section

**Pull request closed without merging**